python/ray/tests/test_iter.py

```diff
@@ -32,6 +33,16 @@ def f(x):
         LocalIterator.get_metrics()
 
 
+def test_zip_with_source_actor(ray_start_regular_shared):
```

Comment:
Can we always place single tests into a unitest.TestCase class?

Reply:
We should be doing the opposite since pytest does better log capture for travis (the rest of Ray tests standardize on the format, the few RLlib tests using TestCase are a historical relic).

Reply:
You'll note the rest of the files in this dir use pytest format.

Reply:
Hmm, yeah, just changed a few in a different PR to use all `unittest.TestCase` :D .
Fair enough. I'll do another bulk PR then to revert these.